example/package.json

```diff
@@ -1,18 +1,14 @@
 {
   "name": "example",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "",
   "main": "./src/publisher.ts",
   "scripts": {
     "build": "tsc",
-    "un-publish": "npm run build && node ./src/content-ops/unencrypted/publish.js",
-    "un-consume": "npm run build && node ./src/content-ops/unencrypted/consume.js",
     "publish": "npm run build && node ./src/content-ops/encrypted/publish.js",
     "consume": "npm run build && node ./src/content-ops/encrypted/consume.js",
     "register-root": "npm run build && node ./src/identity-ops/register-root.js",
     "register": "npm run build && node ./src/identity-ops/register.js",
-    "unregister": "npm run build && node ./src/identity-ops/unregister.js",
-    "unregister-root": "npm run build && node ./src/identity-ops/unregister-root.js",
```

Comment:
removed a few unpublish and unregister examples because those are not really working

Reply:
ok